frontend/packages/text-editor/src/TextRow.tsx

```diff
@@ -136,7 +132,7 @@ export const TextRow = ({
           {showButton && (
             <Button
               aria-label={'toggle-textkey-edit'}
```

Comment:
No big deal, but should we remove the brackets here? I noticed that it was like this before your code changes as well.😊 
```suggestion
              aria-label='toggle-textkey-edit'
```